client/containers/SubredditContainer.js

```diff
@@ -1,13 +0,0 @@
-import {connect} from 'react-redux'
```

Comment:
I like this - I think the idea of containers vs components added more confusion than it helped.